### PR TITLE
added CSI-4446 as new known issue

### PR DESCRIPTION
**Pull request by rivkap**

### CODE REVIEW

File: docs/content/release_notes/known_issues.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ The following severity levels apply to known issues:
...
-|**CSI-3382**|Service|After CSI Topology label deletion, volume provisioning does not work, even when not using any topology-aware YAML files.<br />**Workaround:** To allow volume provisioning through the CSI driver, delete the operator pod. <br />After the deletion, a new operator pod is created and the controller pod is automatically restarted, allowing for volume provisioning.|
-|**CSI-2157**|Service|In extremely rare cases, too many Fibre Channel worker node connections may result in a failure when the CSI driver attempts to attach a pod. As a result, the `Host for node: {0} was not found, ensure all host ports are configured on storage` error message may be found in the IBM block storage CSI driver controller logs. <br />**Workaround:** Ensure that all host ports are properly configured on the storage system. If the issue continues and the CSI driver can still not attach a pod, contact IBM Support.|
+|**CSI-4446**|Service|In rare cases, the HostDefiner labels are not deleted during csinode deletion from the nodes. This occurs even when the `allowDelete` parameter is set to `true`.<br>**Workaround:** Manually delete the hosts from the storage system.
```
Comment (matancarmeli7) [inline docs/content/release_notes/known_issues.md:20]:
```suggestion
|**CSI-4446**|Service|In rare cases, the `hostdefiner.block.csi.ibm.com/manage-node=true` labels are not deleted during csinode deletion from the nodes. This occurs even when the `allowDelete` 
 and `dynamicNodeLabeling` parameters are set to `true`.<br>**Workaround:** Manually delete the hosts from the storage system.
```

Comment (rivkap) [reply docs/content/release_notes/known_issues.md]: updated

File: docs/content/release_notes/known_issues.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ The following severity levels apply to known issues:
...
+|**CSI-4446**|Service|In rare cases, the HostDefiner `hostdefiner.block.csi.ibm.com/manage-node=true` labels are not deleted during csinode deletion from the nodes. This occurs even when the `allowDelete` and `dynamicNodeLabeling` parameters are set to `true`.<br>**Workaround:** Manually delete the hosts from the storage system.
+|**CSI-3382**|Service|After CSI Topology label deletion, volume provisioning does not work, even when not using any topology-aware YAML files.<br>**Workaround:** To allow volume provisioning through the CSI driver, delete the operator pod. <br>After the deletion, a new operator pod is created and the controller pod is automatically restarted, allowing for volume provisioning.|
```
Comment (matancarmeli7) [inline docs/content/release_notes/known_issues.md:21]: could you align the spaces between the issues?

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**